doc/develop/api/overview.rst

```diff
@@ -15,374 +15,301 @@ current :ref:`stability level <api_lifecycle>`.
      - 1.0
-     - 3.1
 
    * - :ref:`kscan_api`
      - Stable
      - 2.1
-     - 2.6
 
    * - :ref:`kernel_api`
      - Stable
      - 1.0
-     - 3.2
 
    * - :ref:`led_api`
      - Stable
      - 1.12
-     - 2.6
 
    * - :ref:`lwm2m_interface`
      - Unstable
      - 1.9
-     - 3.2
 
    * - :ref:`logging_api`
      - Stable
      - 1.13
-     - 3.2
 
    * - :ref:`lora_api`
      - Experimental
      - 2.2
-     - 3.2
 
    * - :ref:`lorawan_api`
      - Experimental
      - 2.5
-     - 3.1
 
    * - :ref:`mbox_api`
      - Experimental
      - 1.0
-     - 3.1
 
    * - :ref:`mqtt_socket_interface`
      - Unstable
      - 1.14
-     - 2.4
 
    * - :ref:`mipi_dsi_api`
      - Experimental
      - 3.1
-     - 3.1
 
    * - :ref:`misc_api`
      - Stable
      - 1.0
-     - 2.2
 
    * - :ref:`networking_api`
      - Stable
      - 1.0
-     - 3.2
 
    * - :ref:`nvs_api`
      - Stable
      - 1.12
-     - 3.2
 
    * - :ref:`peci_api`
      - Stable
      - 2.1
-     - 2.6
 
    * - :ref:`ps2_api`
      - Stable
      - 2.1
-     - 2.6
 
    * - :ref:`pwm_api`
      - Stable
      - 1.0
-     - 3.1
 
    * - :ref:`pinctrl_api`
      - Experimental
      - 3.0
-     - 3.1
 
    * - :ref:`pinmux_api`
      - Stable
      - 1.0
-     - 3.2
 
    * - :ref:`pm_api`
      - Experimental
      - 1.2
-     - 3.2
 
    * - :ref:`random_api`
      - Stable
      - 1.0
-     - 2.1
 
    * - :ref:`regulator_api`
      - Experimental
      - 2.4
-     - 2.4
 
    * - :ref:`reset_api`
      - Experimental
      - 3.1
-     - 3.1
 
    * - :ref:`rtio_api`
      - Experimental
      - 3.2
-     - 3.2
 
    * - :ref:`spi_api`
      - Stable
      - 1.0
-     - 3.2
 
    * - :ref:`sensor_api`
      - Stable
      - 1.2
-     - 3.2
 
    * - :ref:`settings_api`
      - Stable
      - 1.12
-     - 3.2
 
    * - :ref:`shell_api`
      - Stable
      - 1.14
-     - 3.2
 
    * - :ref:`stream_flash`
      - Experimental
      - 2.3
-     - 2.3
 
    * - :ref:`sdhc_api`
      - Experimental
      - 3.1
-     - 3.1
 
    * - :ref:`task_wdt_api`
      - Experimental
      - 2.5
-     - 2.5
 
    * - :ref:`tcpc_api`
      - Experimental
      - 3.1
-     - 3.1
 
    * - :ref:`uart_api`
      - Stable
      - 1.0
-     - 3.1
 
    * - :ref:`UART async <uart_api>`
      - Unstable
      - 1.14
-     - 2.2
 
    * - :ref:`usb_api`
      - Stable
      - 1.5
-     - 3.2
 
    * - :ref:`usbc_api`
      - Experimental
      - 3.3
-     - 3.3
 
    * - :ref:`usermode_api`
      - Stable
      - 1.11
-     - 1.11
 
    * - :ref:`usbc_vbus_api`
      - Experimental
      - 3.3
-     - 3.3
 
    * - :ref:`util_api`
      - Experimental
      - 2.4
-     - 3.2
 
    * - :ref:`video_api`
      - Stable
      - 2.1
-     - 2.6
 
    * - :ref:`w1_api`
      - Experimental
      - 3.2
-     - 3.2
 
    * - :ref:`watchdog_api`
      - Stable
      - 1.0
-     - 2.0
+
+Details about API changes between major releases are available in the :ref:`zephyr_release_notes`.
```

Comment:
I think it'd be better to put this at the top. Otherwise LGTM.

Reply:
Sure thing, moved with the first sentence.